scripts/backups.sh

```diff
@@ -17,15 +17,15 @@ function s3_config() {
 
 # Cleanup S3 bucket
 function clean_s3bucket() {
```

Comment:
Is there a way to taste this?. I think that if we are to include this in tests, how would we be able to do it

Reply:
If you want to test it you should have already an s3 running and a postgres database and just run your container with the right parameters and see